Redesign

skwon02 opened the pull request.

Commit:
🏡💈 Checkpoint
./README.md:21940503/31

Commit:
🦁📻 Checkpoint
./src/_layouts/base.html:21940503/497
./src/index.html:21940503/12

Commit:
🏨📖 Checkpoint
./src/index.html:21940503/324
./src/index copy.html:21940503/261
./src/about copy.html:21940503/261
./src/classes copy.html:21940503/261
./src/registration copy.html:21940503/261
./src/contact copy.html:21940503/261
./src/about.html:21940503/381
./src/contact.html:21940503/254
./src/registration.html:21940503/252
./src/classes.html:21940503/263

Commit:
🌿🏗 Checkpoint
./src/classes.html:21940503/4
./src/contact.html:21940503/15
./src/registration.html:21940503/18
./src/about.html:21940503/2
./src/_data/siteData.json:21940503/92

Commit:
🌑🚞 Checkpoint
./.glitch-assets:21940503/4162
./src/index.html:21940503/2244

Commit:
🏈🐝 Checkpoint
./src/index.html:21940503/1305

Commit:
👀👔 Checkpoint
./src/index.html:21940503/207

Commit:
❣️👠 Checkpoint
./src/index.html:21940503/25

Commit:
😜💖 Checkpoint
./src/index.html:21940503/7191
./src/_layouts/base.html:21940503/205

Commit:
test

Commit:
🛍🌾 Checkpoint
./src/index.html:21940503/2828

Commit:
🐏🏈 Checkpoint

Commit:
🛤🎈 Checkpoint
./src/index.html:21940503/1465

Commit:
👮🌍 Checkpoint
./src/index.html:21940503/24

Commit:
🙉👂 Checkpoint
./src/index.html:21940503/23

Commit:
💍🐽 Checkpoint
./src/index.html:21940503/124

Commit:
🔨😼 Checkpoint
./src/index.html:21940503/53

Commit:
💇🎻 Checkpoint
./src/index.html:21940503/5750

Commit:
🚨🏙 Checkpoint
./src/index.html:21940503/161

Commit:
🌠☔️ Checkpoint
./src/index.html:21940503/16

Commit:
🏉🏯 Checkpoint
./src/_data/siteData.json:21940503/70
./src/index.html:21940503/1330

Commit:
🙌🍞 Checkpoint
./src/index copy.html:21940503/5174
./src/index copy 2.html:21940503/5174
./src/index copy 3.html:21940503/5174
./src/index copy 4.html:21940503/5174
./src/contact copy.html:21940503/5174
./src/about.html:21940503/11
./src/classes.html:21940503/23
./src/register.html:21940503/2
./src/contact.html:21940503/2
./src/login.html:21940503/2

Commit:
💃🍵 Checkpoint
./src/contact.html:21940503/13
./src/login.html:21940503/33
./src/register.html:21940503/32

Commit:
⛲️💏 Checkpoint
./src/login.html:21940503/11803
./src/index.html:21940503/898

Commit:
☎️🌏 Checkpoint
./src/index.html:21940503/2135

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
-<p>Add the HTML for your homepage here.</p>
+<iframe src="https://www.youtube.com/embed/z_jefhcVRAk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="about">
```
In general, at this stage where you are establishing the markup but don't yet know what you'll need to do for styles, I would not include class names yet. Add classes as they are needed in your CSS, but only as needed.

angeliquejw replied:
Consider replacing this `div` with a `section` element?

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fabout.png?v=1605143407347" alt="Dancing Girl">
+    </div>
+    <div class="text">
+        <h1>About MSDC</h1>
```
You have multiple `h1` elements on this page, including the one in `_includes/site-header.html` -- a page should generally only have one `h1` element.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+<iframe src="https://www.youtube.com/embed/z_jefhcVRAk" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<div class="about">
+    <div class="img">
+        <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fabout.png?v=1605143407347" alt="Dancing Girl">
+    </div>
```
You likely don't need this `div` yet -- again, you may end up needing extra markup, but you don't want to start with it.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <button class="more">Learn More</button>
+    </div>
+</div>
+<div class="offerings">
+        <h1>Our CLass Offerings</h1>
```
Same notes about using `section` and not having multiple `h1` elements.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        </li>
+    </ul>
+</div>
+<div class="Register">
+    <h1>2020 Fall Registration</h1>
```
Same notes about using `section` and not having multiple `h1` elements.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    <button class="shedule">Schedule</button>
+    <button class="tuition">Tuition</button>
+</div>
+<div class="contact">
+    <h1>Get In Touch</h1>
```
Same notes about using `section` and not having multiple `h1` elements.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+</div>
+
+<div class="footer">
+    <h1>Morton Street Dance Center</h1>
```
This should not be an `h1`, especially way down here in the footer.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    <button class="subscribe">Subscribe</button>
+</div>
+
+<div class="footer">
```
Consider using the `footer` element instead?

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+
+<div class="footer">
+    <h1>Morton Street Dance Center</h1>
+    <ui class="sitemap">
```
I think you mean to use the `ul` element here and on the closing element for this list.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <li>Registration</li>
+        <li>Contact</li>
+    </ui>
+    <ui class="smedia">
```
I think you mean to use the `ul` element here and on the closing element for this list.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <li>Twitter</li>
+        <li>Pinterest</li>
+    </ui>
+    <ui class="ohour">
```
I think you mean to use the `ul` element here and on the closing element for this list.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <li>Thursday/Friday: 10am-6pm</li>
+        <li>Saturday: 8:30am-2:30pm</li>
+    </ui>
+    <ui class="f-contact">
```
I think you mean to use the `ul` element here and on the closing element for this list.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <h1>Our CLass Offerings</h1>
+    <ul>
+        <li class="ballet">
+            <img src="https://cdn.glitch.com/82a46cd3-6336-42f9-9881-2be3f5a54a0d%2Fballet.png?v=1605143408910" alt="Ballet">
```
Here, the `alt` attribute just repeats the heading that follows, which is not ideal -- this means someone using a screen reader hears "image, ballet, heading, ballet" and the image `alt` attribute isn't providing them any additional information.

Two things to consider:
1. Using a blank `alt` attribute like so: `alt=""`
2. Writing more descriptive `alt` values that convey some detail about the dancers pictured. This would give screen reader users a more equal experience.

This same comment applies to all the `img` elements in this list, but I won't repeat it for each.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    <h1>2020 Fall Registration</h1>
+    <button class="accordion">Step 1</button>
+    <div class="panel">
+        <p>
+            CURRENT STUDENTS: Check your email! You will have class placements sent to you. If you want to add a class you have not taken before, please contact us before registering. <br>
+            PROSPECTIVE STUDENTS: Read our class descriptions, then call or email us to discuss appropriate class placement.
+        </p>
```
These should be separate `p` elements and not use the `br` element.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    </div>
+    <button class="accordion">Step 2</button>
+    <div class="panel">
+        <p>
+            REVIEW CALENDAR, TUITION, REGISTRATION PROCEDURES<br>
+            Double ​check the classes you would like to take. Some classes meet twice per week. Make note of any potential conflicts and talk to us if you have one.
+            Commit to the year or join us for our 13-Week Session! Save money by paying in full. You will otherwise be put on a monthly payment plan.</p>
```
Same note about using multiple `p` elements. Also, here and above, consider using another heading level or a `span` to achieve the all caps text (see [HTML best practices](https://mica-web.github.io/learn/html/best-practices#do-not-use-all-caps-in-your-markup-content-for-presentational-purposes)).

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+
+    <button class="accordion">Step 3</button>
+    <div class="panel">
+        <p>
+            REGISTER ONLINE<br>
+            Log in to your account, or create a new one!
+            Once you've selected your classes, we will review and approve your registration.​ We automatically charge a deposit for registration fee, September tuition, and costume deposit(s) to secure your space in the classes you've selected. You will then be sent a confirmation email and balance due for the season.</p>
```
Same note as above for this content. I would expect to see something like this instead:
```suggestion
        <h3>Register Online</h3>
        <p>Log in to your account, or create a new one!</p>
        <p>Once you've selected your classes, we will review and approve your registration.​ We automatically charge a deposit for registration fee, September tuition, and costume deposit(s) to secure your space in the classes you've selected. You will then be sent a confirmation email and balance due for the season.</p>
```

(The heading level may be different depending on the structure you end up with here; this assumes the `h1` for this section becomes an `h2`).

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    </div>
+    <button class="accordion">Step 4</button>
+    <div class="panel">
+        <p>
+            ORDER YOUR UNIFORMS AND MSDC GEAR<br>
+             We have created a convenient list of approved, affordable attire for classes through Curtain Call for Class​. You can also order MSDC gear through this link! All students are required to be in uniform; Repertory students must also purchase a sweatsuit.</p>
```
Same note about all caps, `p` and `br`.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+            ORDER YOUR UNIFORMS AND MSDC GEAR<br>
+             We have created a convenient list of approved, affordable attire for classes through Curtain Call for Class​. You can also order MSDC gear through this link! All students are required to be in uniform; Repertory students must also purchase a sweatsuit.</p>
+    </div>
+    <button class="shedule">Schedule</button>
+    <button class="tuition">Tuition</button>
```
What do these `button` elements do when they are clicked?

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+</div>
+<div class="Register">
+    <h1>2020 Fall Registration</h1>
+    <button class="accordion">Step 1</button>
```
All accordion `button` elements must have a set `type` attribute, like so:
```suggestion
    <button class="accordion" type="button">Step 1</button>
```

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    <h1>Get In Touch</h1>
+    <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3086.1248737884016!2d-76.6446764842207!3d39.330756228887225!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89c81b2abe742d9b%3A0xa8f579926b1381e!2sMorton%20Street%20Dance%20Center!5e0!3m2!1sen!2sus!4v1605202579180!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen="" aria-hidden="fals ...
+    <form>
+        <label for="fname">First name:</label><br>
+        <input type="text" id="fname" name="fname"><br>
+
+        <label for="lname">Last name:</label><br>
+        <input type="text" id="lname" name="lname"><br>
+
+        <label for="email">Email:</label><br>
+        <input type="text" id="email" name="email"><br>
+
+        <label for="phone">Phone:</label><br>
+        <input type="text" id="phone" name="phone"><br>
+
+        <label for="message">Type your message here...</label><br>
+        <input type="text" id="message" name="message"><br>
```
Remove all these `br` elements as the layout will be controlled via CSS instead.

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+        <input type="text" id="lname" name="lname"><br>
+
+        <label for="email">Email:</label><br>
+        <input type="text" id="email" name="email"><br>
```
There's a separate input `type` for emails 👍🏻
```suggestion
        <input type="email" id="email" name="email"><br>
```

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+
+        <input type="submit" value="Submit">
+      </form>
+    <button class="tour">Request a Tour</button>
+    <button class="app">Download Our App</button>
+    <button class="subscribe">Subscribe</button>
```
Can you describe what these buttons do when clicked?

angeliquejw commented on src/index.html:
```diff
@@ -5,4 +5,151 @@
+    </ui>
+    <ui class="f-contact">
+        <li>Tel. 410-235-9003</li>
+        <li>Email. information@mortonstreetdance.com</li>
```
You can make this email easily clickable for users:

```suggestion
        <li>Email. <a href="mailto:information@mortonstreetdance.com">information@mortonstreetdance.com</a></li>
```

angeliquejw reviewed (commented):
Hi, Soyeon!

I left some notes about how to improve the HTML here, plus some questions about some of the buttons/interactions to make sure I understand them clearly.

Let me know if you have questions about these recommendations!

Commit:
🎉🌁 Checkpoint
./src/index.html:21940503/46

Commit:
🍟👾 Checkpoint
./src/index.html:21940503/593

Commit:
🚅🛋 Checkpoint
./src/index.html:21940503/624

Commit:
💡🚠 Checkpoint
./src/index.html:21940503/155

Commit:
🌓🌥 Checkpoint
./src/index.html:21940503/1560

Commit:
⏰🐾 Checkpoint
./src/index.html:21940503/1003
./src/_includes/site-footer.html:21940503/906
./src/register.html:21940503/5134
./src/login.html:21940503/5345
./src/contact.html:21940503/5162
./src/classes.html:21940503/5145
./src/about.html:21940503/15402

Commit:
🚉✋ Checkpoint
./src/_layouts/base.html:21940503/190

Commit:
🐞👩‍👩‍👦 Checkpoint
./src/index.html:21940503/723

Commit:
⛲️📙 Checkpoint
./src/index.html:21940503/377

Commit:
⛹🎋 Checkpoint
./src/index.html:21940503/76

Commit:
🏬💄 Checkpoint
./src/sass/components/_components.scss:21940503/556
./src/sass/components/_home.scss:21940503/194
./src/sass/layout/_site-nav.scss:21940503/61
./src/sass/_variables.scss:21940503/375
./src/sass/_base.scss:21940503/236

Commit:
👭🐗 Checkpoint
./src/sass/_variables.scss:21940503/213
./src/sass/_base.scss:21940503/511
./src/sass/layout/_site-nav.scss:21940503/1011

Commit:
🎑🌲 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/354
./src/sass/_variables.scss:21940503/121

Commit:
🏥⛪️ Checkpoint
./src/sass/layout/_site-nav.scss:21940503/193
./src/index.html:21940503/65
./src/about.html:21940503/2
./src/classes.html:21940503/2
./src/register.html:21940503/2
./src/contact.html:21940503/4
./src/sass/layout/_site-header.scss:21940503/664

Commit:
🙅⛱ Checkpoint
./src/sass/layout/_site-nav.scss:21940503/136
./src/sass/layout/_site-header.scss:21940503/234

Commit:
🤘🍓 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/40
./src/sass/layout/_site-header.scss:21940503/3

Commit:
💴🐵 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/624
./src/index.html:21940503/59
./src/about.html:21940503/2
./src/classes.html:21940503/2
./src/register.html:21940503/2

Commit:
🤑🕋 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/575
./src/sass/layout/_site-header.scss:21940503/51
./src/sass/layout/_layout.scss:21940503/6
./src/_layouts/default.html:21940503/9
./src/sass/layout/_site-footer.scss:21940503/2762
./src/login.html:21940503/1

Commit:
🚎✊ Checkpoint
./src/index.html:21940503/5305
./src/classes.html:21940503/5178
./src/about.html:21940503/832

Commit:
🕋🍚 Checkpoint
./src/index.html:21940503/11758
./src/sass/components/_home.scss:21940503/12

Commit:
🚡🚜 Checkpoint
./src/sass/components/_home.scss:21940503/28
./src/index.html:21940503/212
./src/sass/utilities/_button.scss:21940503/136
./src/sass/components/_components.scss:21940503/13

Commit:
👘🍇 Checkpoint
./src/sass/components/_home.scss:21940503/201
./src/sass/_variables.scss:21940503/56
./src/index.html:21940503/259

Commit:
👏🛥 Checkpoint
./src/sass/components/_home.scss:21940503/380
./src/index.html:21940503/6

Commit:
👫🐦 Checkpoint
./src/sass/components/_home.scss:21940503/335
./src/index.html:21940503/5

Commit:
🐟👵 Checkpoint
./src/index.html:21940503/159
./src/sass/components/_home.scss:21940503/940
./src/sass/_base.scss:21940503/24

Commit:
🛰🙉 Checkpoint
./src/sass/components/_home.scss:21940503/34

Commit:
🏘🌃 Checkpoint
./src/sass/components/_button.scss:21940503/23
./src/index.html:21940503/278
./src/sass/components/_home.scss:21940503/707

Commit:
💽🍈 Checkpoint
./src/sass/components/_home.scss:21940503/86
./src/index.html:21940503/102
./src/sass/components/_button.scss:21940503/2351

Commit:
🏐🏉 Checkpoint
./src/sass/components/_button.scss:21940503/2274
./src/index.html:21940503/189

Commit:
🛥🗾 Checkpoint
./src/sass/components/_button.scss:21940503/402
./src/sass/components/_home.scss:21940503/35
./src/sass/layout/_site-footer.scss:21940503/127

Commit:
🚒👩 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/758
./src/_includes/site-footer.html:21940503/7

Commit:
🎟💺 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/369

Commit:
🌦⛄️ Checkpoint
./src/sass/components/_home.scss:21940503/66

Commit:
🏡🎇 Checkpoint
./src/sass/components/_components.scss:21940503/63

Commit:
👜🐯 Checkpoint
./src/sass/components/_home.scss:21940503/4233
./src/index.html:21940503/2084

Commit:
☕️🙎 Checkpoint
./.glitch-assets:21940503/7344

Commit:
🚲🍜 Checkpoint
./src/index.html:21940503/1879

Commit:
🦄🍨 Checkpoint
./src/index.html:21940503/605

Commit:
⛪️🍚 Checkpoint
./.glitch-assets:21940503/2937
./src/index.html:21940503/650

Commit:
✌️🍍 Checkpoint
./src/index.html:21940503/1776
./.glitch-assets:21940503/4068

Commit:
🐽🚇 Checkpoint
./src/index.html:21940503/5772

Commit:
🚆🚈 Checkpoint
./src/about.html:21940503/12197
./.glitch-assets:21940503/2304

Commit:
🎅🏡 Checkpoint
./src/_includes/site-nav.html:21940503/906
./menu-bundled.js:21940503/8046
./menu-w-trap.js:21940503/1050
./menu.js:21940503/925
./src/sass/components/_menu-icon.scss:21940503/723
./src/sass/layout/_site-nav.scss:21940503/532
./src/sass/utilities/_list-reset.scss:21940503/187

Commit:
🏁🌨 Checkpoint
./src/_includes/site-nav.html:21940503/177
./src/_layouts/base.html:21940503/205

Commit:
🏟💶 Checkpoint
./src/sass/components/_home.scss:21940503/303
./src/index.html:21940503/66

Commit:
🐓💕 Checkpoint
./.glitch-assets:21940503/5064
./src/index.html:21940503/2330
./src/sass/components/_components.scss:21940503/56

Commit:
👝🚘 Checkpoint
./src/_layouts/base.html:21940503/42
./src/_includes/site-nav.html:21940503/118
./src/index.html:21940503/78

Commit:
🍩🍧 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/1040

Commit:
🚗🏚 Checkpoint

Commit:
⏱🚗 Checkpoint
./src/sass/components/_about.scss:21940503/72
./src/sass/components/_components.scss:21940503/29

Commit:
❣️🚉 Checkpoint
./src/sass/components/_about.scss:21940503/53
./src/about.html:21940503/55

Commit:
🌆🚵 Checkpoint
./src/sass/components/_about.scss:21940503/451
./src/about.html:21940503/611

Commit:
🍩👟 Checkpoint
./src/sass/components/_about.scss:21940503/483
./src/about.html:21940503/46

Commit:
🐑🐝 Checkpoint
./src/sass/components/_about.scss:21940503/188

Commit:
👫🍬 Checkpoint
./src/sass/components/_about.scss:21940503/425

Commit:
⛄️🛩 Checkpoint
./src/sass/components/_about.scss:21940503/228

Commit:
🙍⏰ Checkpoint
./src/sass/components/_about.scss:21940503/846
./src/about.html:21940503/28

Commit:
☺️👬 Checkpoint
./src/about.html:21940503/1388
./src/sass/components/_about.scss:21940503/410
./src/index.html:21940503/14
./src/sass/components/_home.scss:21940503/44

Commit:
⛹🚗 Checkpoint
./src/sass/components/_home.scss:21940503/44
./src/about.html:21940503/410
./src/index.html:21940503/14
./src/sass/components/_about.scss:21940503/880
./.glitch-assets:21940503/382

Commit:
⛽️🐢 Checkpoint
./src/about.html:21940503/2071
./src/sass/components/_home.scss:21940503/4

Commit:
🍖🚓 Checkpoint
./src/about.html:21940503/562
./src/sass/components/_about.scss:21940503/2137
./src/slideshow.js:21940503/738
./src/index.html:21940503/39

Commit:
🎸🍟 Checkpoint
./src/_includes/site-nav.html:21940503/39
./src/_layouts/base.html:21940503/42
./src/index.html:21940503/158
./src/sass/components/_home.scss:21940503/383

Commit:
🐂🌌 Checkpoint
./src/sass/components/_home.scss:21940503/162
./src/index.html:21940503/535

Commit:
🏝🏖 Checkpoint
./src/sass/components/_home.scss:21940503/12
./src/index.html:21940503/100

Commit:
🌚🦁 Checkpoint
./src/index.html:21940503/4237
./src/sass/components/_home.scss:21940503/106
./src/accordion.js:21940503/1638
./src/accordion-alt.js:21940503/2538

Commit:
👨‍❤️‍👨💕 Checkpoint
./src/js/base.js:21940503/239
./src/_layouts/base.html:21940503/9

Commit:
🎷🍜 Checkpoint
./src/_layouts/base.html:21940503/21

Commit:
👡👩 Checkpoint
./src/_layouts/base.html:21940503/29
./.eleventy.js:21940503/50

Commit:
🍩👔 Checkpoint
./src/sass/components/_home.scss:21940503/18

Commit:
🏊🏼💴 Checkpoint
./src/sass/components/_home.scss:21940503/254
./src/sass/components/_menu-icon.scss:21940503/51

Commit:
🌒👓 Checkpoint
./src/sass/components/_menu-icon.scss:21940503/248
./src/sass/layout/_site-nav.scss:21940503/29

Commit:
💓⚽️ Checkpoint
./src/sass/layout/_site-nav.scss:21940503/175
./src/sass/components/_menu-icon.scss:21940503/17

Commit:
🍏⚽️ Checkpoint
./src/sass/components/_menu-icon.scss:21940503/87
./src/sass/components/_home.scss:21940503/1630
./src/sass/components/_grid.scss:21940503/1825
./src/sass/components/_components.scss:21940503/13

Commit:
😄🏮 Checkpoint
./src/sass/components/_home.scss:21940503/382
./src/js/menu.js:21940503/1789

Commit:
🎏🎍 Checkpoint
./src/js/accordion copy.js:21940503/2820
./src/js/menu.js:21940503/6096
./src/index.html:21940503/57
./src/js/accordion-alt.js:21940503/2
./src/sass/components/_home.scss:21940503/238

Commit:
💧👦 Checkpoint
./src/index.html:21940503/80
./src/js/accordion.js:21940503/2
./src/js/form.js:21940503/99
./src/sass/components/_components.scss:21940503/8

Commit:
❄️👄 Checkpoint
./src/index.html:21940503/379
./src/sass/components/_form.scss:21940503/1033

Commit:
🌎🚋 Checkpoint
./src/sass/components/_form.scss:21940503/109
./src/index.html:21940503/12

Commit:
🐋🐪 Checkpoint
./src/sass/components/_form.scss:21940503/372

Commit:
💰🌌 Checkpoint
./src/sass/components/_form.scss:21940503/260

Commit:
🏉😇 Checkpoint
./src/index.html:21940503/90
./src/sass/components/_form.scss:21940503/22
./src/sass/components/_button.scss:21940503/235
./src/sass/components/_grid.scss:21940503/68

Commit:
💡🌁 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/95
./src/_includes/site-footer.html:21940503/122

Commit:
💋📉 Checkpoint
./src/_includes/site-footer.html:21940503/700
./src/sass/layout/_site-footer.scss:21940503/1207

Commit:
🗺🎗 Checkpoint
./src/_includes/site-footer.html:21940503/70
./src/sass/layout/_site-footer.scss:21940503/9

Commit:
🏗🧀 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/76
./src/sass/components/_button.scss:21940503/8

Commit:
👠🐪 Checkpoint
./src/sass/components/_grid.scss:21940503/589

Commit:
🏢💈 Checkpoint
./src/sass/components/_grid.scss:21940503/651
./src/sass/components/_form.scss:21940503/142

Commit:
🏁🚣 Checkpoint
./src/sass/components/_form.scss:21940503/280

Commit:
🎓👦 Checkpoint
./src/sass/components/_form.scss:21940503/235
./src/index.html:21940503/10
./src/sass/components/_home.scss:21940503/224

Commit:
🐩🖇 Checkpoint
./src/sass/components/_home.scss:21940503/33

Commit:
🤑🎓 Checkpoint
./src/sass/components/_home.scss:21940503/687

Commit:
💅🍮 Checkpoint
./src/sass/components/_home.scss:21940503/186

Commit:
🎩📅 Checkpoint
./src/sass/components/_home.scss:21940503/385

Commit:
🎤👂 Checkpoint
./src/sass/components/_home.scss:21940503/250

Commit:
🎓🐷 Checkpoint
./src/sass/components/_home.scss:21940503/393

Commit:
🏚👏 Checkpoint
./src/sass/components/_home.scss:21940503/272

Commit:
🏹👚 Checkpoint
./src/sass/components/_home.scss:21940503/45
./src/index.html:21940503/191

Commit:
💓🌇 Checkpoint
./src/index.html:21940503/794
./src/sass/components/_home.scss:21940503/32
./src/js/menu.js:21940503/1735
./src/_layouts/base.html:21940503/18

Commit:
🏹💨 Checkpoint
./src/sass/components/_home.scss:21940503/550

Commit:
🐔🏢 Checkpoint
./src/sass/components/_home.scss:21940503/412
./src/sass/components/_button.scss:21940503/8

Commit:
✋💪 Checkpoint
./src/sass/components/_button.scss:21940503/8
./src/sass/components/_home.scss:21940503/19
./src/_layouts/minimal.html:21940503/38

Commit:
👴☺️ Checkpoint
./src/sass/layout/_site-footer.scss:21940503/76

Commit:
💾⏳ Checkpoint
./src/sass/layout/_site-footer.scss:21940503/540

Commit:
🏥🏇 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/1068

Commit:
👴🏡 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/621
./src/_includes/site-footer.html:21940503/538

Commit:
🎾🏖 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/210

Commit:
🍩😋 Checkpoint
./src/login.html:21940503/13
./src/sass/layout/_site-nav.scss:21940503/8

Commit:
🙉🗺 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/85

Commit:
📼🌼 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/56

Commit:
🏈👨‍👨‍👦 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/1019
./src/sass/components/_menu-icon.scss:21940503/58

Commit:
👳🚝 Checkpoint
./src/sass/components/_menu-icon.scss:21940503/46
./src/sass/layout/_site-nav.scss:21940503/93
./src/index.html:21940503/86

Commit:
🎋🌖 Checkpoint
./src/index.html:21940503/123

Commit:
👌🚓 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/7
./src/index.html:21940503/753

Commit:
💡👹 Checkpoint
./src/_includes/site-footer.html:21940503/180
./src/index.html:21940503/171
./src/sass/layout/_site-nav.scss:21940503/45

Commit:
🏵🤗 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/93

Commit:
🍜👣 Checkpoint
./src/sass/components/_button.scss:21940503/4

Commit:
☄️🏐 Checkpoint
./src/_includes/slide.html:21940503/209
./src/_includes/slideshow-icons.html:21940503/1178
./src/_includes/slideshow-nav.html:21940503/1510
./src/sass/components/_slideshow.scss:21940503/2632
./src/sass/components/_components.scss:21940503/15
./src/js/slide.js:21940503/5938

Commit:
🐣🍖 Checkpoint
./src/about.html:21940503/530
./src/_data/siteData.json:21940503/1032

Commit:
😚💝 Checkpoint
./src/_data/siteData.json:21940503/1555

Commit:
📠👬 Checkpoint
./src/_layouts/base.html:21940503/90
./src/sass/components/_slideshow.scss:21940503/96
./src/_data/siteData.json:21940503/1361

Commit:
🎠📙 Checkpoint
./src/_data/testimonials.json:21940503/4634
./src/about.html:21940503/334
./src/_includes/slide.html:21940503/111

Commit:
📚🏆 Checkpoint
./.glitch-assets:21940503/911
./src/register.html:21940503/3114

Commit:
😇🎺 Checkpoint
./.glitch-assets:21940503/381
./src/classes.html:21940503/6649
./src/404.html:21940503/383
./src/sass/components/404.scss:21940503/1
./src/sass/components/_components.scss:21940503/11
./src/sass/components/_404.scss:21940503/252

Commit:
💘😘 Checkpoint
./src/404.html:21940503/334
./src/sass/components/_404.scss:21940503/1231

Commit:
🤖⚾️ Checkpoint
./src/sass/components/_404.scss:21940503/999
./src/404.html:21940503/25

Commit:
💰🌜 Checkpoint
./src/sass/components/_404.scss:21940503/211
./src/sass/_base.scss:21940503/35

Commit:
🌥💆 Checkpoint
./.glitch-assets:21940503/857
./src/classes.html:21940503/190
./src/register.html:21940503/193
./src/404.html:21940503/8
./src/about.html:21940503/8
./src/sass/_base.scss:21940503/276
./src/sass/layout/_site-footer.scss:21940503/35

Commit:
🚏🏄 Checkpoint
./src/404.html:21940503/8
./src/sass/components/_about.scss:21940503/1314
./src/about.html:21940503/58

Commit:
🍋👢 Checkpoint
./src/sass/components/_about.scss:21940503/901
./src/about.html:21940503/269
./src/sass/components/_404.scss:21940503/64

Commit:
📮🐦 Checkpoint
./src/sass/components/_about.scss:21940503/716
./src/about.html:21940503/11

Commit:
🎹🐇 Checkpoint
./src/sass/components/_about.scss:21940503/207

Commit:
🚿🏓 Checkpoint
./src/sass/components/_home.scss:21940503/226

Commit:
☔️🗾 Checkpoint
./src/index.html:21940503/127
./src/sass/components/_button.scss:21940503/135
./src/sass/components/_home.scss:21940503/764

Commit:
🕷🍽 Checkpoint
./src/index.html:21940503/94
./src/sass/components/_home.scss:21940503/337
./src/register.html:21940503/5034
./src/sass/components/_form.scss:21940503/49

Commit:
🐞🚌 Checkpoint
./src/sass/components/_home.scss:21940503/195
./src/sass/components/_form.scss:21940503/167
./src/index.html:21940503/466

Commit:
👛🔋 Checkpoint
./src/index.html:21940503/10
./src/sass/components/_button.scss:21940503/204
./src/sass/components/_about.scss:21940503/201
./src/sass/components/_grid.scss:21940503/31

Commit:
🐃🌪 Checkpoint
./src/about.html:21940503/95
./src/sass/components/_grid.scss:21940503/1271

Commit:
🚧⛵️ Checkpoint
./src/_includes/slideshow-nav.html:21940503/20
./src/js/menu_accordion.js:21940503/18713

Commit:
✌️🚙 Checkpoint
./src/_data/testimonials.json:21940503/2
./src/sass/components/_slideshow.scss:21940503/353
./src/about.html:21940503/26

Commit:
🎖💈 Checkpoint
./src/_includes/slide.html:21940503/261
./src/sass/components/_slideshow.scss:21940503/462
./src/_includes/slideshow-nav.html:21940503/50

Commit:
👨🍰 Checkpoint
./src/sass/components/_slideshow.scss:21940503/183
./src/sass/components/_home.scss:21940503/5

Commit:
🐆🐴 Checkpoint
./src/_data/class.json:21940503/101

Commit:
📼😜 Checkpoint
./src/_data/class.json:21940503/5521

Commit:
👗🚛 Checkpoint
./src/_data/class.json:21940503/1459
./src/index.html:21940503/389

Commit:
🐫👙 Checkpoint
./src/classes.html:21940503/86
./src/index.html:21940503/2345
./src/_includes/grid.html:21940503/3041

Commit:
✈️🏑 Checkpoint
./src/classes.html:21940503/2285
./src/_includes/grid.html:21940503/1350
./src/_data/class.json:21940503/934
./src/sass/components/_grid.scss:21940503/727

Commit:
🛤🚌 Checkpoint
./src/sass/components/_grid.scss:21940503/190

Commit:
☎️🎌 Checkpoint
./src/sass/components/_grid.scss:21940503/51

Commit:
🗾🦃 Checkpoint
./src/sass/components/_grid.scss:21940503/2400

Commit:
🛫📟 Checkpoint
./src/about.html:21940503/20

Commit:
🚐🎊 Checkpoint
./src/_includes/grid.html:21940503/498
./src/sass/components/_grid.scss:21940503/1020
./src/js/menu_accordion.js:21940503/4763

Commit:
🎣🎨 Checkpoint
./src/_includes/grid.html:21940503/6613
./src/sass/components/_grid.scss:21940503/542

Commit:
👔🚖 Checkpoint
./src/sass/components/_grid.scss:21940503/11
./src/_includes/grid.html:21940503/1249

Commit:
😋👋 Checkpoint
./src/index.html:21940503/544
./src/sass/components/_button.scss:21940503/145

Commit:
✨😋 Checkpoint
./src/index.html:21940503/7
./src/sass/components/_button.scss:21940503/215

Commit:
🚨✌️ Checkpoint
./src/_includes/grid.html:21940503/38
./src/index.html:21940503/92
./src/_includes/site-nav.html:21940503/721
./src/sass/layout/_site-nav.scss:21940503/130

Commit:
🏤🎅 Checkpoint
./src/_includes/site-nav.html:21940503/206
./src/index.html:21940503/2891
./src/_includes/accordion.html:21940503/2704
./src/register.html:21940503/2544
./src/sass/layout/_site-nav.scss:21940503/19

Commit:
🏌✨ Checkpoint
./src/sass/layout/_site-nav.scss:21940503/141

Commit:
🤘👠 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/6
./src/_includes/site-nav.html:21940503/9
./src/sass/components/_home.scss:21940503/7
./src/index.html:21940503/75

Commit:
🐬⚡️ Checkpoint
./src/_includes/slide.html:21940503/5
./src/_data/testimonials.json:21940503/133
./src/index.html:21940503/1190
./src/sass/layout/_site-header.scss:21940503/852
./src/sass/components/_form.scss:21940503/340

Commit:
🐴🏈 Checkpoint
./src/sass/components/_form.scss:21940503/1360
./src/index.html:21940503/44

Commit:
🏩🍨 Checkpoint
./src/sass/layout/_site-footer.scss:21940503/93
./src/sass/components/_form.scss:21940503/19

Commit:
🍼🗺 Checkpoint
./src/sass/components/_home.scss:21940503/95
./src/_includes/site-nav.html:21940503/452

Commit:
🎰💏 Checkpoint
./src/_includes/site-nav.html:21940503/135
./src/sass/layout/_site-nav.scss:21940503/377

Commit:
🚃🌨 Checkpoint
./src/_includes/site-nav.html:21940503/130
./src/sass/layout/_site-nav.scss:21940503/395

Commit:
🛩🗺 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/516

Commit:
🚪🦂 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/313

Commit:
👸👨‍❤️‍👨 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/941

Commit:
🌨😼 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/1940
./src/sass/components/_home.scss:21940503/64

Commit:
🐦🍫 Checkpoint
./src/sass/components/_home.scss:21940503/284
./src/sass/components/_button.scss:21940503/2
./src/sass/components/_form.scss:21940503/2

Commit:
🛎🚀 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/303

Commit:
🏐😂 Checkpoint
./src/sass/components/_slideshow.scss:21940503/5

Commit:
🎍📠 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/7

Commit:
🚡🙌 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/127

Commit:
🎟📍 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/43
./src/_includes/site-nav.html:21940503/126

Commit:
🐊🌂 Checkpoint
./src/_includes/site-nav.html:21940503/133
./src/sass/layout/_site-nav.scss:21940503/259
./src/index.html:21940503/9

Commit:
🌌🐡 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/880
./src/about.html:21940503/18
./src/_includes/site-nav.html:21940503/19

Commit:
🏔🚥 Checkpoint
./src/sass/layout/_site-nav.scss:21940503/262

Commit:
😂🕷 Checkpoint
./src/_includes/site-nav.html:21940503/719

Commit:
🚣😂 Checkpoint
./src/_includes/site-nav.html:21940503/125
./src/sass/layout/_site-nav.scss:21940503/306

Commit:
🏤🎇 Checkpoint
./src/sass/layout/_site-header.scss:21940503/407
./src/sass/layout/_site-nav.scss:21940503/14

skwon02 commented:
# Project 4
Your Glitch project URL: https://glitch.com/edit/#!/project4-redesign

## Grading option
Choose **one** of the following by marking it with a ✅:

- [V] I want detailed feedback on my GitHub PR
_this option is similar to how I have graded your previous projects_

- [ ] I do not require comments on GitHub, just a grade
_selecting this option, you will get a grading rubric sheet, but no comments on GitHub_

## Long(er) answers

> If you had another week or two to work on this project, what else would you do?
First of all, I want to add alt texts for all images and buttons. Next, I would like to have more detailed pages and a fixed button for registration that is rotating all the time. There are a few pages that I couldn't make such as each class page, log in, contact page. The fixed button would help the students and parents find the registration easily.

>  Describe which aspect of this project you found most confusing or frustrating.

Definitely the Eleventy. I've got used to finding my files at the end, but it was so confusing in the beginning. Connecting each file and controlling the CSS made up of Eleventy was difficult to google. Also, the gap between my design and coding skill made me frustrating. I had more plans but I had to give up a few designs corresponding to my coding skill. It was a great experience to better understand developers though.

> Share what about this project you are most proud of.

I'm proud of the fact that I just made this website. Although the code is not the best, it was a good experience to learn the process from evaluation to design and developing. I like my grid the most. I enjoyed making it. 

> What is the most beneficial thing you've learned in this course?

In addition to coding languages, it was great to learn about multiple aspects of building a website: UX, design, and code. Also, the most important part was to learn about an inclusive website. I will try to apply this to my portfolio website.

## Short(er) answers

> Summarize the project requirements and evaluate if you have met them.

The MSDC is a responsive website, using Sass. It has a homepage and 404 pages, including about, classes, and registration as inner pages. The interactive elements are the contact form, slides for testimonials, and the accordion for registration steps.

> Did you review the best practices checklists for [HTML](https://mica-web.github.io/learn/html/best-practices) and [CSS](https://mica-web.github.io/learn/css/best-practices) to check your own work on this project?

Yes, but I wasn't able to add alt texts to images.

Commit:
🍈🎓 Checkpoint
./src/sass/_variables.scss:21940503/84
